website/content/api-docs/secret/consul.mdx

```diff
@@ -70,18 +70,20 @@ $ curl \
 
 This endpoint creates or updates the Consul role definition. If the role does
 not exist, it will be created. If the role already exists, it will receive
-updated attributes.
+updated attributes. At least one of `consul_policies`, `consul_roles`,
+`service_identities`, or `node_identities` is required depending on the
+Consul version.
 
 | Method | Path                  |
 | :----- | :-------------------- |
 | `POST` | `/consul/roles/:name` |
 
 ### Parameters for Consul versions 1.11 and above
 
-- `partition` `(string: "")` <EnterpriseAlert inline /> - Specifies the Consul admin partition the token
-  will be generated within. The partition must exist, and the policies or roles assigned to the
-  Vault role must also exist inside the given partition. If not provided, the "default"
-  partition is used.
+- `partition` `(string: "")` - Specifies the Consul admin partition in which the token is generated within.
```

Comment:
"In which" means "where" so it already specifies location. Ending the sentence in "within" is redundant and awkward. The proper usage should be one of the following:
- Specifies the Consul admin partition **in which** the token is generated.
- Specifies the Consul admin partition **which** the token is generated **within**.

Personally, I prefer the first.

See
- https://linguaholic.com/linguablog/how-to-use-in-which-in-a-sentence/
- https://www.usingenglish.com/forum/threads/in-which.142245/#post-722936

Reply:
I think we should also add "default" to the type info:
```suggestion
- `partition` `(string: "default")` - Specifies the Consul admin partition in which the token is generated.
```

Reply:
Good catch!